From c41293e4953e990675f3f5f2741cc612d921db7d Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Thu, 20 Feb 2025 11:40:53 +0100
Subject: [PATCH] disable unittests

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec2822542c0f..51a69410b848 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -498,9 +498,6 @@ if(BUILD_PYTHON_INTERFACE)
     add_subdirectory(python)
   endif()
 endif()
-if(BUILD_TESTING)
-  add_subdirectory(test)
-endif(BUILD_TESTING)
 
 PKG_CONFIG_APPEND_LIBS("coal")
 if(COAL_HAS_OCTOMAP)
-- 
2.53.0

